PixelHunter1995/InventoryLib/InventoryItem.cs

```diff
@@ -0,0 +1,31 @@
+﻿
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using PixelHunter1995.TilesetLib;
+
+namespace PixelHunter1995.InventoryLib
+{
+    class InventoryItem
```

Comment:
I think it might be a good idea to put this in the same file as Inventory.

Reply:
I don't think sooo